acceptance/repository-postgresql/README.md

```diff
@@ -0,0 +1,66 @@
+# @loopback/test-repository-postgresql
+
+Acceptance tests for `@loopback/repository` + `loopback-connector-postgresql`.
+
+## Running the test suite
+
+### Using own PostgreSQL instance
+
+If you have a local PostgreSQL instance listening on `localhost` and the default
+port, with a `root` user and an empty password, use the following command:
+
+```bash
+npm test
+```
+
+If you have a local or remote PostgreSQL instance and would like to use that to
+run the test suite, use the following command:
+
+**Linux & MacOS**
+
+```bash
+POSTGRESQL_HOST=<HOST> POSTGRESQL_PORT=<PORT> POSTGRESQL_USER=<USER> POSTGRESQL_PASSWORD=<PASSWORD> POSTGRESQL_DATABASE=<DATABASE> npm test
+```
+
+**Windows**
+
+```bash
+SET POSTGRESQL_HOST=<HOST>
+SET POSTGRESQL_PORT=<PORT>
+SET POSTGRESQL_USER=<USER>
+SET POSTGRESQL_PASSWORD=<PASSWORD>
+SET POSTGRESQL_DATABASE=<DATABASE>
+npm test
+```
+
+### Using Docker (Linux, MacOS, WSL)
+
+If you do not have a local PostgreSQL instance, you can also run the test suite
+with very minimal requirements.
+
+- Assuming you have [Docker](https://docs.docker.com/engine/installation/)
+  installed, run the following script which would spawn a PostgreSQL instance on
+  your local:
+
+  ```bash
+  source setup.sh <HOST> <PORT> <USER> <PASSWORD> <DATABASE>
+  ```
+
+  Where `<HOST>`, `<PORT>`, `<USER>`, `<PASSWORD>` and `<DATABASE>` are optional
+  parameters. The default values are `localhost`, `5432`, `root`, `pass` and
+  `testdb` respectively.
```

Comment:
I think `testdb` is not accurate, because we use `repository_tests`? How about the second database, `repository_tests_new`?

Reply:
I think when you're running it locally via the setup script, it uses `testdb`. It only creates `repository_tests` if the environment variable isn't set through the setup script. Also, this is in line with [`mysql`](https://github.com/strongloop/loopback-next/commit/e9dca4cbcc8ebf4e2567cebdc046f3cf1d886060#diff-7ec3ca7c559cb1be02f07d4bf8ccef34R51).

Reply:
I'll update this to `repository_tests` as it's set up in `schema.sql`